### ISSUE TITLE
Broken tier 1 links

### ISSUE THREAD

**Issue by DiamondMonster090** [P1xt/p1xt-guides#206]
I tried to look at the descriptions of tier 1, as I am steadily progressing through tier 0 and wanted to know what I am going to do, and I saw that 3 links are broken:
1) The software debugging course now leads to a 404 as seen by this image below:
![image](https://github.com/P1xt/p1xt-guides/assets/120121712/9e057435-5912-4b95-894c-bd256a3aa220)
2) The software testing course instead now leads to a "Site reliability engineer nanodegree" as seen by this screenshot:
![image](https://github.com/P1xt/p1xt-guides/assets/120121712/1f6bffcf-a3e3-4cd7-835e-8328b9dfd7be)
3) In the practice section, google kick start 2018 also shows a 404, as as seen by this image:
![image](https://github.com/P1xt/p1xt-guides/assets/120121712/a981730d-ee5a-4866-8501-4334e7e7f421)
Is the nanodegree the correct thing? or is it also a broken link?
Thank you!

**Comment by P1xt** [comment]
I'm a little busy but I'll take a look. At this point. I'd recommend just skipping those three in Tier 1 in the meantime. It looks like Udacity may have changed all their links, and I'll need to find the correct ones or see if there's a better resource available, as it's been a couple of years since I put those in to add a bit of breadth to the learning.

**Comment by DiamondMonster090** [comment]
As I continued looking through the guide I saw more broken links, a lot of them from Google competitions that are no longer hosted on their website. I found a github repository with an archive of all Google coding competitions, which maybe could be used instead of the broken links. [Here](https://github.com/google/coding-competitions-archive) is the link to the repo: https://github.com/google/coding-competitions-archive

**Comment by DiamondMonster090** [comment]
For your information, the other broken links I found (searching the whole guide) were: outdated react and angular docs, mongoDB courses leading to mongoDB university homepage and not the course page, some udacity courses are broken (I would try to find the exact ones), the xamarin docs are outdated, the Odin project ruby link is broken, and the geeks4ggeks quizzes.
I am sorry! I know that this is a lot of work, but after I found some broken links in section one, I just had some more time and I searched the next sections. In the next days I would try to see if all these courses were deleted or just links changed, and hopefully try to help fix it.
